assets/src/ImageBlockExtension.js

```diff
@@ -92,25 +92,25 @@ const addExtraControls = function() {
               title={ __( 'Planet4 Image Options' ) }
               initialOpen={ true }
             >
-              <label>Caption alignment</label>
+              <label htmlFor>Caption alignment</label>
               <ButtonGroup>
-              {
-                captionAlignmentOptions.map((option, key) => {
-									return <Button
-										key={key}
-										value={option.value}
-										onClick={() => {
-											props.setAttributes({
-                        captionAlignment: option.value
-                      });
-                      updateCaptionAlignment(option.value);
-										}}
-										isPrimary={ captionAlignment === option.value }
-										isSecondary={ captionAlignment !== option.value }>
-											{ option.label }
-									</Button>;
-                })
-              }
+                {
+                  captionAlignmentOptions.map((option, key) => {
+                    return <Button
+                      key={key}
+                      value={option.value}
+                      onClick={() => {
+                        props.setAttributes({
+                          captionAlignment: option.value,
+                        });
+                        updateCaptionAlignment(option.value);
+                      }}
+                      isPrimary={ captionAlignment === option.value }
+                      isSecondary={ captionAlignment !== option.value }>
```

Comment:
Thinking loud, why we leave an empty space between `{` and its value, but in other places we don't use that rule? 
For example [here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/1006/files#diff-28439af7370575789c096bf0633de7a826ac204b2f6341ee7115939709452ca3R100)
`value={option.value}` but `isPrimary={ captionAlignment === option.value }`

Reply:
Oh, nice catch, strange it didn't return an error 🤔